common/meta/embedded/src/meta_embedded.rs

```diff
@@ -89,7 +89,7 @@ impl MetaEmbedded {
         common_meta_sled_store::init_sled_db(path);
 
         {
-            let mut m = GLOBAL_META_EMBEDDED.as_ref().lock().unwrap();
+            let mut m = GLOBAL_META_EMBEDDED.as_ref().lock().await;
```

Comment:
https://rust-lang.github.io/rust-clippy/master/index.html#await_holding_lock

so, i change to `tokio::sync::Mutex`